README.md

```diff
@@ -1,58 +1,237 @@
+
+- **Scalability**: Alfred is designed to scale with your needs, whether you're processing thousands of documents a day or millions.
+
 ## Features
 
 - **Comprehensive Authentication Support**: Seamlessly handles OAuth, HMAC, and API key authentication methods, simplifying the process of connecting to the Alfred API.
 - **Domain-Specific Operations**: Offers specialized support for File and Job operations, enabling developers to intuitively manage and interact with API resources.
 - **Cross-Platform Compatibility**: Designed to be fully compatible across .NET Core, .NET Standard, and .NET Framework 4.7.2, ensuring broad usability in diverse development environments.
 - **Minimal Dependencies**: Crafted to minimize external dependencies, facilitating an easier integration and deployment process with reduced conflict risk.
 
-## Getting Started
-
-### Prerequisites
+## Prerequisites
 
 - .NET Core 2.1+ or .NET Framework 4.7.2+ installed on your development machine.
-- An active Alfred API key for authentication.
+- An active Alfred API key, HMAC credentials, or OAuth credentials for authenticating API requests.
 
-### Installation
+## Installation
 
 To integrate the Alfred API wrapper into your .NET project, install the package via NuGet:
 
 ```bash
 dotnet add package TagShelf.Alfred.ApiWrapper
 ```
 
-### Usage
+Alternatively, you can install the package using the NuGet Package Manager in Visual Studio. Search for `TagShelf.Alfred.ApiWrapper` and click the Install button.
 
-1. **Initialize the Client**
+For .NET Core CLI, use the following command:
 
-   Begin by creating an instance of the Alfred client using your preferred authentication method:
+```bash
+Install-Package TagShelf.Alfred.ApiWrapper
+```
+
+## Usage
+
+### Initialize the Client
+
+   Begin by creating an instance of the Alfred client using your preferred authentication method along with the desired environment type (Production or Staging). The following examples demonstrate how to initialize the client with different authentication methods:
+
+#### Environments description
+
+- **Production**: The production environment are used for live applications and real-world scenarios. It is recommended to use this environment for production-ready applications. The frontend URL for the production environment is [https://app.tagshelf.com](https://app.tagshelf.com)</br></br>
+
+- **Staging**: The staging environment is used for testing and development purposes. It is recommended to use this environment for testing and development scenarios. The frontend URL for the staging environment is [https://staging.tagshelf.com](https://staging.tagshelf.com)</br></br>
+
+#### Authentication Methods
+
+The following examples demonstrate how to initialize the Alfred client with different authentication methods:
+
+   For OAuth authentication, specify the method and credentials explicitly and provide the environment type (Production or Staging):
```

Comment:
Where do I get my credentials from? By answering this question I think we cover this:

> 2. Where do I start so I can use this library that allows me to access Alfred?